appcenter-analytics/Analytics.d.ts

```diff
@@ -1,6 +1,6 @@
 export function trackEvent(
   eventName: string,
   properties?: { [name: string]: string }
-): Promise<string | null>;
+): Promise<null>;
```

Comment:
You used void instead of null in other places, since we are actually returning undefined, it should not be null but void.